### ISSUE TITLE
Create login.gov adoption dashboard to track key metrics

### ISSUE THREAD

**Issue by elizabethmkoch** [department-of-veterans-affairs/va.gov-team#50658]
## Issue Description
The login.gov adoption team, stakeholders, and VA leadership don't have a way to easily view the information surrounding VA identity and access methods and key metrics. 

Potential metrics to track:
- number of existing accounts by cred
- Number of sign ins by week and month
- Proofed/not proofed
- Migration segments (target number for inherited proofing, grandfathering, other)
- Number of new accounts being created for all creds
- Track direct email comms (open rates, bouncebacks, correlate to inherited proofing / grandfather flow or other directions)

## High Level User Story/ies
As a VA stakeholder, I need visibility into login metrics so I can track login.gov adoption progress.

## Acceptance Criteria
- [ ] Key metrics are able to be seen on a dashboard
- [ ] Key metrics are added to a report and sent to stakeholders

## Hypothesis or Bet
By providing a dashboard that enables stakeholders to visualize key metrics, stakeholders will have better insight into the progress of the login.gov adoption, help identify where improvements need to be made, and will be able to make better decisions to address any gaps.

## We will know we're done when... ("Definition of Done")
*What requirements does this project need to meet for you to finish this initiative?*
- [ ] Critical login CSP metrics are identified and exposed through dashboard
- [ ] We have identified and created a data set for obtaining metrics

## Known Blockers/Dependencies 
*List any blockers or dependencies for this work to be completed*
- Subject matter expertise in Domo and/or DataDog
- Training from Analytics and Insights Team
- Data from IAM 

## Projected Launch Date
Q2 2023

*Take into consideration Accessibility/QA needs as well as Product, Technical, and Design requirements.*

**Comment by tygindraux** [comment]
In the collab meeting with the Core team, Joe noted that it's worth reviewing the metrics we've decided to capture as a team. Since this work has been primarily led by Product and Engineering, I wanted to make sure we also reviewed this from a UX perspective. I captured my thoughts here: https://docs.google.com/document/d/1s52atGNWwSTZQAOqlqIldrLg2h2pLaBAVfmBUdCr0-Q/edit?usp=sharing

Jay put in a meeting on Monday to discuss.
